### PR TITLE
fixed rare edge case which could result in undefined value in end of …

### PR DESCRIPTION
**Pull request by cliff-bohm**
…run, time to coalescence, report. This is a follow up to the previous PR to close (hopefully) one last hole.

### CODE REVIEW

File: Archivist/LODwAPArchivist/LODwAPArchivist.cpp [hunk 1]
```diff
@@ -204,16 +202,25 @@ bool LODwAPArchivist::archive(std::vector<std::shared_ptr<Organism>> &population
...
+		  else if (time_to_coalescence > 0) {
+			  std::cout << "Time to Coalescence for the last organism written to LOD_data.csv (at time " << dataSequence[data_seq_index - 1] << ") is "
+				  << time_to_coalescence << "." << std::endl;
+		  }
+		  else if (time_to_coalescence == 0) {
+			  std::cout << "The last organism written to LOD_data.csv (at time " << dataSequence[data_seq_index - 1] << ") had a time to coalescence of 0.\n" <<
+				  "Rejoice! This organism is on LOD!" << std::endl;
+		  }
+		  else {
+			  std::cout << "The last organism written to LOD_data.csv had a negitive time to coalescence.\n" <<
```
Comment (cgnitash) [inline Archivist/LODwAPArchivist/LODwAPArchivist.cpp]: If this is a true error, then we should exit(1), instead of a clean exit.

**Comment by cliff-bohm** [comment]
I don't think exit(1) would be correct here. This is only reporting the status at the very end of the run. It is possible that a -1 resulted from a sync issue and not an issue with the actual result. An exit(1) here would cause the run to terminate just before the last generations organisms were written to disk - which either would be bad because there was no problem - or because, if there was, we don't have that data to help debug.

**Comment by cliff-bohm** [comment]
Alternatively, I think we could probably safely remove that clause. The way the code is now, I don't think a -1 is possible.

**Comment by cgnitash** [comment]
I'm fine with either removing the clause if it's unreachable, or causing a very visible error if that clause is triggered.

File: Archivist/LODwAPArchivist/LODwAPArchivist.cpp (hunk 1 above)
Comment (JorySchossau) [reply Archivist/LODwAPArchivist/LODwAPArchivist.cpp]: I agree that removing the clause or making a very visible error message is good.

**Comment by cgnitash** [comment]
Looks good.

**Pull request merged**